pkg/k8sclient/k8sclient.go

```diff
@@ -67,10 +63,22 @@ func GetNode(nodeName string, kubeClient kubernetes.Interface) (*v1.Node, error)
 }
 
 func UpdateNodeStatus(nodeName string, kubeClient kubernetes.Interface, node *v1.Node) error {
-	updatenode, err := kubeClient.CoreV1().Nodes().UpdateStatus(node)
+	_, err := kubeClient.CoreV1().Nodes().UpdateStatus(node)
 	if err != nil {
 		return err
 	}
-	log.Printf("DEBUG: UpdateNodeStatus: %+v\n", updatenode)
 	return nil
 }
+
+func GetLocalStorageClassName(kubeClient kubernetes.Interface) (string, error){
+	scList, err := kubeClient.StorageV1().StorageClasses().List(metav1.ListOptions{})
```

Comment:
in the example under :https://kubernetes.io/docs/concepts/storage/persistent-volumes/#storage-object-in-use-protection the Pv has a type=local label on it
is it automatically added to it?
if not, can't we make the static provisioner to add some labels?
it would be better to label our own PVs so we recognize them, rather than needing to read the SC API all the time